translator/translate/otel/receiver/otlp/translator_test.go

```diff
@@ -217,3 +217,41 @@ func TestTranslateAppSignals(t *testing.T) {
 		})
 	}
 }
+
+func TestTranslateJMX(t *testing.T) {
+	tt := NewTranslatorWithName(common.JmxKey, WithDataType(component.DataTypeMetrics))
+	testCases := map[string]struct {
+		input   map[string]interface{}
+		want    *confmap.Conf
+		wantErr error
+	}{
+		"WithJMXEnabled": {
+			input: map[string]interface{}{
+				// @TODO fill this out after pipeline is built
```

Comment:
why?

Reply:
Then is no validation for the values in the map yet